samples/MultiBoundTypeVariableUnspecToObject.java

```diff
@@ -51,17 +51,17 @@ <T extends Object & Lib> Object x0(@NullnessUnspecified T x) {
   }
 
   <T extends @Nullable Object & Lib> Object x6(@NullnessUnspecified T x) {
-    // :: error: jspecify_nullness_not_enough_information jspecify_but_expect_error
+    // :: error: jspecify_nullness_not_enough_information
     return x;
   }
 
   <T extends @Nullable Object & @NullnessUnspecified Lib> Object x7(@NullnessUnspecified T x) {
-    // :: error: jspecify_nullness_not_enough_information jspecify_but_expect_error
+    // :: error: jspecify_nullness_not_enough_information
     return x;
   }
 
   <T extends @Nullable Object & @Nullable Lib> Object x8(@NullnessUnspecified T x) {
-    // :: error: jspecify_nullness_mismatch
+    // :: error: jspecify_nullness_not_enough_information
```

Comment:
```suggestion
    // :: error: jspecify_nullness_mismatch jspecify_but_expect_warning
```

Under our current substitution rules, I think that `jspecify_nullness_mismatch` is correct: `@NullnessUnspecified T` can currently "add" nullness to the given type but not "remove" nullness from it. So, if `T` is a nullable type (as it can be, thanks to the bounds), `@NullnessUnspecified T` can be nullable, too.

This may change when we look at https://github.com/jspecify/jspecify/issues/248 again, but for now, it's what our rules say, and it's what other samples are set up for.

Reply:
(Err, I was assuming above that the checker was currently producing a warning—hence "jspecify_but_expect_warning." But my main point is about why I think "jspecify_nullness_mismatch" is correct. Then we can fill in an "override" like "jspecify_but_expect_warning" or "jspecify_but_expect_nothing" if the checker currently has one of those behaviors and we want to encode it here.)

Reply:
I've undone this change.